pipeline/compilers/__init__.py

```diff
@@ -90,18 +86,39 @@ def is_outdated(self, infile, outfile):
 
 
 class SubProcessCompiler(CompilerBase):
-    def execute_command(self, command, content=None, cwd=None):
+    def execute_command(self, command, content=None, cwd=None, stdout_as_result=None):
```

Comment:
``stdout_as_result`` is quite obscure, and doesn't really reflect the actual behavior.